lib/iex/test/iex/interaction_test.exs

```diff
@@ -19,6 +19,11 @@ defmodule IEx.InteractionTest do
     assert capture_iex("1 + 2") == "3"
   end
 
+  test "quoted operators" do
```

Comment:
It may not be the right place t o test this, but Inspect.Algebra does not have any related test